src/main/java/com/github/jonatino/process/DataSource.java

```diff
@@ -27,6 +27,8 @@
 public interface DataSource {
 
 	MemoryBuffer read(Pointer address, int size);
+	
+	MemoryBuffer read(Pointer address, int size, MemoryBuffer tobuf);
```

Comment:
Okay, here are the screenshots: http://imgur.com/a/o9FLG

Btw, there's a new update for CS:GO - new map, new case, new player models! ^_^